src/vue/Alert.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <div
+    v-if="show"
+    transition="fade"
+    class="uk-alert"
+    :class="{
+      'uk-alert-success': style === 'success',
+      'uk-alert-warning': style === 'warning',
+      'uk-alert-danger': style === 'danger',
+      'uk-alert-large': size === 'large'
+    }">
+    <a href="" class="uk-close uk-alert-close" v-if="close" @click.prevent="toggleMessage()"></a>
+    <slot></slot>
+  </div>
+</template>
+
+<script>
+import UI from 'uikit'
+import Vue from 'vue'
+
+const transitionOptions = {
+  duration: 200,
+  fade: true
+}
+
+export default {
+  props: {
+    style: {
+      type: String,
+      default: ''
+    },
+    size: {
+      type: String,
+      default: ''
+    },
+    close: {
+      type: Boolean,
+      default: true
+    },
+    show: {
+      type: Boolean,
+      default: true
+    },
+    transition: {
+      type: Object,
+      default: function () {
+        return transitionOptions
+      },
+      coerce: function (val) {
+        return Object.assign(transitionOptions, val)
+      }
+    }
+  },
+  methods: {
+    toggleMessage () {
+      this.show = !this.show
+    }
+  },
+  transitions: {
+    fade: {
+      css: false,
+      leave: function (el, done) {
+        if (this.transition.fade) {
+          UI.$(el).animate({opacity: 0}, this.transition.duration, done)
+        } else {
+          Vue.util.remove(el)
```

Comment:
If we remove the element it can be reused to show another message. Instead just set show to false